src/Sandbox.Core/Database.cs

```diff
@@ -50,4 +50,12 @@ private T OpenConnection()
         return Activator.CreateInstance(typeof(T), _connectionString) as T
                ?? throw new Exception("Failed to create a db connection");
     }
+
+    public async Task LogData(string id, string data, DateTime timestamp)
+    {
+        using var connection = OpenConnection();
+        const string sql = "INSERT INTO PlantData (Timestamp,McuID,Data) "
+                           + "VALUES (Timestamp, @ID, @Data)";
+        await connection.ExecuteAsync(sql, new {ID = id, Timestamp = timestamp, Data = data });
```

Comment:
Keep params in the same order as it is easy to end up with bugs where data is written to the wrong column.
```suggestion
    public async Task LogData(string id, string data, DateTime timestamp)
    {
        using var connection = OpenConnection();
        const string sql = "INSERT INTO PlantData (McuID, Data, Timestamp) "
                           + "VALUES (@ID, @Data, @Timestamp)";
        await connection.ExecuteAsync(sql, new { ID = id, Timestamp = timestamp, Data = data });
```